server/app/views/applicant/ApplicantInformationView.java

```diff
@@ -80,7 +81,7 @@ public Content render(
 
     String submitText = messages.at(MessageKey.BUTTON_UNTRANSLATED_SUBMIT.getKeyName());
     ButtonTag formSubmit =
-        submitButton(submitText).withClasses(ApplicantStyles.BUTTON_SELECT_LANGUAGE);
+        submitButton(submitText).withClasses(ButtonStyles.SOLID_BLUE, "text-base", "mx-auto");
```

Comment:
Looks like SOLID_BLUE already has the "text-base" class?

Reply:
Done.